### PR TITLE
create oldcontent directory and move reviewed items there

### PR DESCRIPTION
**Pull request by sharifX**
Reviewed old folders and pages. Created oldcontent directory and moved the following items there: 
- apispec/
- faqs
- json-examples-and-schemas
- ods-ontology/
- templates (this was for creating git issues)
- serialization.md

updated README to introduce openDS as a specification and remove old content. simplified readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,106 +1,16 @@
-# openDS
-The home of the specification for open Digital Specimens (openDS).
+open Digital Specimen (openDS) is the cornerstone of DiSSCo's vision for a seamless and standardised digital representation of physical specimens in Natural Science Collections. openDS serves as a comprehensive data specification, unifying diverse scientific information derived from both biological and geological specimens. Born out of the need for a structured description that integrates effortlessly into various workflows, openDS is designed to enhance the Findable, Accessible, Interoperable, and Reusable (FAIR) aspects of biodiversity data.
```
Comment (TomDijkema) [inline README.md:1]: Capital O.

File: README.md [hunk 2]
```diff
@@ -1,106 +1,16 @@
...
-- The openDS Application Programming Interface (API). Read the [introduction to the openDS API](api-spec/api-intro.md).
-
-## Moving forward
-
-Specification and agreement of openDS is an iterative, spiral process of continuous development (implementation) and improvement (specification) that can begin with the building of one or more 'Hello World' kind of demonstrators. One such example is the demonstrator at [nsidr.org](https://nsidr.org/) / [demo.nsidr.org](https://demo.nsidr.org/). Lessons can be learned from such implementations and fed into the specification process.
+As a digital surrogate on the internet, openDS encapsulates the essence of a "Digital Specimen," offering a solution for collections ranging from marine specimens to botanical and mineralogical holdings. Leveraging established biodiversity information standards such as Darwin Core, Minimal Information Digital Specimen, Latimer Core, Access to Biological Collection Data Schema, Extension for Geosciences, and the emerging Global Biodiversity Information Facility Unified Model, openDS harmonises and elevates the quality and machine actionability of scientific data.
```
Comment (TomDijkema) [inline README.md:3]: for in Extension for Geosciences should also be capitalized to: For (as its EFG)

File: README.md [hunk 3]
```diff
@@ -1,106 +1,16 @@
...
-
-This page of [use cases](Use-cases.md) explains some of the scenarios of use envisaged for openDS.
-
-## Resources
-
-*To be added.*
-
-
-END.
+As of 2024, DiSSCo is supported by different EU grants. We have a dedicated development team and other collaborators working towards a robust specification. You can see some of the implementations in our current [Sandbox](https://sandbox.dissco.tech/).
```
Comment (TomDijkema) [inline README.md:14]: Everything is written in in formal, so I would reformat the second sentence to: A dedicated development team and other collaborators work towards a robust specification.

File: README.md [hunk 4]
```diff
@@ -1,106 +1,16 @@
...
-
-## Resources
-
-*To be added.*
-
-
-END.
+As of 2024, DiSSCo is supported by different EU grants. We have a dedicated development team and other collaborators working towards a robust specification. You can see some of the implementations in our current [Sandbox](https://sandbox.dissco.tech/). 
 
+Linking third-party/extended specimen data together, as openDS proposes, requires many stakeholders to test and adopt the specification. This is especially the case to integrate data at multiple levels: specimen, collection types, species, etc., and cross-discipline to geology, genomics, etc. This necessitates global collaboration among all players in the landscape converging towards a single specification as far as possible.
```
Comment (TomDijkema) [inline README.md:16]: The etc. in between makes it a bit fuzzy to read, maybe this is for keeping geology apart from biology, but then write it a bit differently.

Maybe change to: This is especially the case for integrating data at multiple levels, like: specimen, collection types, species, genomics, and cross-discipline to geology, etc.

**Review by TomDijkema — COMMENTED** [review]
Looks good Sharif! Got a view comments about the text.

Maybe we can take the old content off GitHub sometime in the future if it does not serve much purpose anymore?

File: README.md (hunk 1 above)
Comment (sharifX) [reply README.md]: fixed.

File: README.md (hunk 2 above)
Comment (sharifX) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (sharifX) [reply README.md]: good suggestion. updated the line.

**Comment by sharifX** [comment]
renamed oldcontent to archive. we can delete this folder later, as it will be easier to delete rather that the single files.

**Pull request merged**